src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeMortars.hpp

```diff
@@ -177,6 +169,39 @@ struct InitializeMortars {
         "Dependencies not fulfilled. Did you forget to terminate the phase "
         "after removing options?");
   }
+
+ private:
+  template <typename DbTagsList>
+  static auto initialize_next_temporal_id_on_mortars(
+      db::DataBox<DbTagsList>&& box, std::true_type /* meta */) noexcept {
+    using next_temporal_id_tag =
+        typename BoundaryScheme::receive_temporal_id_tag;
+    using mortars_next_temporal_id_tag =
+        ::Tags::Mortars<next_temporal_id_tag, dim>;
+    const auto& next_temporal_id = get<next_temporal_id_tag>(box);
+    db::item_type<mortars_next_temporal_id_tag> mortar_next_temporal_ids{};
+    // Since no communication needs to happen for boundary conditions
+    // the temporal id is not advanced on the boundary, so we only need to
+    // initialize it on internal boundaries
+    for (const auto& direction_and_neighbors :
+         db::get<domain::Tags::Element<dim>>(box).neighbors()) {
+      const auto& direction = direction_and_neighbors.first;
+      const auto& neighbors = direction_and_neighbors.second;
+      for (const auto& neighbor : neighbors) {
+        const auto mortar_id = std::make_pair(direction, neighbor);
+        mortar_next_temporal_ids.insert({mortar_id, next_temporal_id});
+      }
+    }
+    return ::Initialization::merge_into_databox<
+        InitializeMortars, db::AddSimpleTags<mortars_next_temporal_id_tag>>(
+        std::move(box), std::move(mortar_next_temporal_ids));
+  }
+
+  template <typename DbTagsList>
+  static db::DataBox<DbTagsList> initialize_next_temporal_id_on_mortars(
+      db::DataBox<DbTagsList>&& box, std::false_type /* meta */) noexcept {
+    return std::forward<db::DataBox<DbTagsList>>(box);
```

Comment:
just return box

Reply:
> warning: local variable 'box' will be copied despite being returned by name [-Wreturn-std-move]
> note: call 'std::move' explicitly to avoid copying

So I made this `std::move(box)`. Is that the correct way to forward the argument?